modules/plugins/wfly-10/src/main/java/org/rhq/modules/plugins/wildfly10/DatasourcesComponent.java

```diff
@@ -20,16 +20,8 @@
 package org.rhq.modules.plugins.wildfly10;
 
 import static java.lang.Boolean.TRUE;
-import static org.rhq.core.domain.resource.CreateResourceStatus.FAILURE;
-import static org.rhq.core.domain.resource.CreateResourceStatus.INVALID_ARTIFACT;
-import static org.rhq.core.domain.resource.CreateResourceStatus.SUCCESS;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.CONNECTION_PROPERTIES_ATTRIBUTE;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.DISABLE_OPERATION;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.ENABLED_ATTRIBUTE;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.ENABLE_OPERATION;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.XA_DATASOURCE_PROPERTIES_ATTRIBUTE;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.getConnectionPropertiesAsMap;
-import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.isXADatasourceResource;
+import static org.rhq.core.domain.resource.CreateResourceStatus.*;
+import static org.rhq.modules.plugins.wildfly10.DatasourceComponent.*;
```

Comment:
Le 11/03/2016 21:59, Michael Burman a écrit :
> You've forgot Eclipse settings, no * imports.

Sorry I could not find the setting in IDEA's Eclipse Formatter plugin. 
Do you remember where I can configure this?